src/plugins/path/README.md

```diff
@@ -24,9 +25,45 @@ is allowed to occur for both device and mountpoint. When checking for
 relative files, it is not enough to look at the first character if it is
 a `/`, because remote file systems and some special names are valid, too.
 
+If `check/permission = [permission], [user]` is also present it will check for the correct permissions
+of the file/directory. `check/permission = rw, tomcat` for example will check if the user `tomcat` has read and 
+write access to the path which was set in `check/path`.
+ 
+ Permissions available:
+ - `r`: **R**ead
+ - `w`: **W**rite
+ - `x`: e**X**ecute
+
```

Comment:
I will leave this open for now and see in my thesis if it is needed that often. But the idea is good!

Reply:
Yes, making everything complete is not a good idea anyway. It easily happens that stuff is added nobody ever needs. I just mentioned it so that you are aware that such a check might be relevant for some software.